routes/index.js

```diff
@@ -17,4 +18,6 @@ router.get('/disconnect', AuthController.getDisconnect);
 
 router.get('/users/me', UsersController.getMe);
 
+router.get('/files', FilesController.postUpload);
```

Comment:
the http method is supposed to be 'post'. its `router.post('/files', FileController.postUpload);`

weldone.

Reply:
sorry about that, it was an oversight. i didnt know that i used GET instead of POST